test/unit/org/apache/cassandra/guardrails/GuardrailConsistencyTest.java

```diff
@@ -177,45 +186,43 @@ public void testUpdateWithAllowedConsistency()
         // test that it does not throw
         update(ConsistencyLevel.LOCAL_QUORUM);
 
-        disableConsistencyLevels(SERIAL_ONLY);
+        disableConsistencyLevels(new LinkedHashSet<>(Arrays.asList(ConsistencyLevel.SERIAL.toString())));
```

Comment:
In each case, when using `disableConsitencyLevels`, it would be better (and cleaner) to use just `ImmutableSet.of(a, b, c)` instead of `new LinkedHashSet(Arrays.asList(a, b, c))`. `ImmutableSet.of` preserves order of elements. Also, `ImmutableSet.of` will be no op when called on instance of `ImmutableSet.of` (which happens inside `disableConsistencyLevels`, so using `ImmutableSet.of` here will be just a single collection creation instead of 3